testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java

```diff
@@ -0,0 +1,203 @@
+package xxxxxx.yyyyyy.zzzzzz.selenium.error;
```

Comment:
- [ ] Overall, there is a mix of spaces and tabs in the indentation.
Use https://github.com/terasolunaorg/terasoluna-gfw/blob/master/eclipse/formatter.xml to format.

Reply:
I reapplied everything.